prow/config/config.go

```diff
@@ -813,7 +813,7 @@ func parseProwConfig(c *Config) error {
 	return nil
 }
 
-func (c *Config) decorationRequested() bool {
+func (c *JobConfig) decorationRequested() bool {
```

Comment:
why is this in this PR?

Reply:
because I've rebased off https://github.com/kubernetes/test-infra/pull/8519, https://github.com/kubernetes/test-infra/pull/8521 and https://github.com/kubernetes/test-infra/pull/8543, this is still a valid change